src/PlatformCompat.Analyzers/Fixes/ReportIssueCodeFixProvider.cs

```diff
@@ -0,0 +1,94 @@
+using PlatformCompat.Analyzers.Deprecated;
+using PlatformCompat.Analyzers.Exceptions;
+using PlatformCompat.Analyzers.ModernSdk;
+using PlatformCompat.Analyzers.Net461;
+
+namespace PlatformCompat.Analyzers.Fixes
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(ReportIssueCodeFixProvider))]
+    [Shared]
+    public class ReportIssueCodeFixProvider : CodeFixProvider
+    {
+        public sealed override ImmutableArray<string> FixableDiagnosticIds
+        {
+            get
+            {
+                var result = new List<string>();
+                result.Add(ExceptionAnalyzer.DiagnosticId);
+                result.Add(ModernSdkAnalyzer.DiagnosticId);
+                result.Add(Net461Analyzer.DiagnosticId);
+                result.AddRange(DeprecatedAnalyzer.GetDescriptors().Select(d => d.Id));
+                return result.ToImmutableArray();
+            }
+        }
+
+        public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            var diagnostic = context.Diagnostics.First();
+            var diagnosticSpan = diagnostic.Location.SourceSpan;
+
+            var title = $"{diagnostic.Id}: {diagnostic.GetMessage()}";
+            var action = new ReportIssueCodeAction(title);
+            context.RegisterCodeFix(action, diagnostic);
+
+            return Task.CompletedTask;
+        }
+        
+        private sealed class ReportIssueCodeAction : CodeAction
+        {
+            private readonly string _issueTitle;
+
+            public ReportIssueCodeAction(string issueTitle)
+            {
+                _issueTitle = issueTitle;
+            }
+
+            public override string Title => Resources.ReportAnIssueTitle;
+
+            public override string EquivalenceKey => Title;
+
+            protected override Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(CancellationToken cancellationToken)
+            {
+                var issueTitleEncoded = WebUtility.UrlEncode(_issueTitle);
+                var url = $"https://github.com/dotnet/platform-compat/issues/new?title={issueTitleEncoded}";
+                var result = new[] { new StartProcessCodeOperation(url) };
+                return Task.FromResult<IEnumerable<CodeActionOperation>>(result);
+            }
+        }
+
+        private sealed class StartProcessCodeOperation : CodeActionOperation
+        {
+            public StartProcessCodeOperation(string fileName)
+            {
+                FileName = fileName;
+            }
+
+            public string FileName { get; }
+
+            public override void Apply(Workspace workspace, CancellationToken cancellationToken)
+            {
+                var processClass = Type.GetType("System.Diagnostics.Process, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089");
+                var processClassTypeInfo = processClass.GetTypeInfo();
+                var startMethod = processClassTypeInfo
```

Comment:
When we move to CoreFX 2.0 this will be broken because the default for `UseShellExecute` is `false`. You can use the overload that takes a `ProcessStartInfo` or add a issue so we fix it later.

Reply:
Good point. I should have mentioned that all of this is a giant hack to avoid having to retarget to .NET Standard 1.6. I've now bitten the bullet and synced everything to .NET Framework 4.6.1 and .NET Standard 2.0.

> When we move to CoreFX 2.0 this will be broken because the default for `UseShellExecute` is `false`.

I'll try but I suspect the reason I can open the browser is because I'm using `ShellExecute`. The fixer will not be called in build server scenarios, but it might be called from VS Code (Mac/Linux) or Rider (Mac/Linux). Both of which will run on Mono. No idea whether that will work, but we can easily test that.

Reply:
> bitten the bullet and synced everything to .NET Framework 4.6.1 and .NET Standard 2.0
👍 
Regarding sheel execute on Mac/Linux, Toub's words: "UseShellExecute=true is enabled on Unix in 2.0; it's just more robust/complete post-2.0"